atlas/scripts/utils/fasta.py

```diff
@@ -54,7 +54,12 @@ def split(fasta_file,maxSubsetSize,out_dir, simplify_headers=True):
     SubsetSize= int(ceil( N/ceil(N/ maxSubsetSize)))
     extension= os.path.splitext(fasta_file)[-1]
 
-    os.makedirs(out_dir)
+    # Check whether the first output file already exists.
+    if (os.path.isfile(os.path.join(out_dir, "subset1" + extension)) = True):
+        print(os.path.join(out_dir, "subset1" + extension) + "already exists. Exiting...")
+        sys.exit(1)
+
+    os.makedirs(out_dir, exist_ok=True)
```

Comment:
Previous comment from @SilasK : "Could you add a check that the file subset_1.fast (I'm not shure of the exact spelling)) doesn't exist in this folder.

I want to escape the case that it halve overwrites these files."

Reply:
@SilasK : Done. The code now checks for if the first expected output file already exists.